crates/typst-cli/src/world.rs

```diff
@@ -26,6 +26,11 @@ use crate::package;
 static STDIN_ID: LazyLock<FileId> =
     LazyLock::new(|| FileId::new_fake(VirtualPath::new("<stdin>")));
 
+/// Static `FileId` allocated for empty.
+/// This is to ensure that a file is read in the correct way.
+static EMPTY_ID: LazyLock<FileId> =
```

Comment:
I assume the idea of this is that we need a `SystemWorld` to properly compile the `statement`, but at the same time it requires a main document and there's not really a way around that?

I think "Static `FileId` allocated for empty." is a bit obtuse, maybe this could be better explained here.

Reply:
Yes, the assumption is correct. I thought about implementing a new `World` struct, but that seemed way to repetitive because most of the functionality of `SystemWorld` would still be needed.

In the end, adding another `FileInput` special case seemed to be the easiest, least intrusive solution.

I will elaborate on the comments though.